CutVariations/runCutVar.sh

```diff
@@ -0,0 +1,79 @@
+# \author Alice Caluisi <alice.caluisi@cern.ch>
+# \since December 2023
+
+#
+# This task performs cut variation studies on trees analysis results using the configuration produced by configCutVar.sh
+# The produced results are called AnalysisResults_cutName.root and are stored in Results/LHC_Test/CutVariations folder 
+#
+# To run the task, type: $> bash runCutVar.sh
+
+#!/bin/bash
+
+reset
+CfgDir=$(dirname "$0")
+Cfg="-b"
+
+#Define an array of all the cuts
+cuts=(
+    "maxChi2PerClusterITS=30"
+    "maxChi2PerClusterITS=40"
+    "maxChi2PerClusterTPC=6"
+    "maxChi2PerClusterTPC=4"
+    "globalTrack=true"
+    "minPt=0.2"
+)
+
+#Call the configCutVar.sh script to generate the configuration file
+bash configCutVar.sh "${cuts[@]}"
+
+#Create subdirectory for cut variation studies in Results/LHC_Test folder
+Results="../Results/LHC_Test/"
```

Comment:
is the path to the results still a command line argument, or will you always adjust it for the not LHC_Test dummy ?